docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -16,35 +16,37 @@ Which browser is used depends on:
 
 The following table shows which browser is used for the various platforms and operating systems.
 
-|**OS / Platform**|**Browser**|
+|**OS**|**Office version**|**Edge WebView2 (Chromium-based) installed**|**Browser**|
 |:-----|:-----|:-----|:-----|:-----|:-----|:-----|
-|Office on the web|The browser in which Office is opened.|
-|Mac|Safari|
-|iOS|Safari|
-|Android|Chrome|
-|Windows / non-subscription Office 2013 or later|Internet Explorer 11|
-|Windows 10 ver. < 1903 / Office 365|Internet Explorer 11|
-|Windows 10 ver. >= 1903 / Office 365 ver < 16.0.11629<sup>1</sup>|Internet Explorer 11|
-|Windows 10 ver. >= 1903 / Office 365 ver >= 16.0.11629<sup>1</sup>|Microsoft Edge<sup>2, 3</sup>|
+|any|Office on the web|N/A|The browser in which Office is opened.|
+|Mac|any|N/A|Safari|
+|iOS|any|N/A|Safari|
+|Android|any|N/A|Chrome|
+|Windows 7, 8.1, 10 | non-subscription Office 2013 or later|Doesn't matter|Internet Explorer 11|
+|Windows 7 | Microsoft 365| Doesn't matter | Internet Explorer 11|
+|Windows 8.1,<br>Windows 10 ver.&nbsp;<&nbsp;1903| Microsoft 365 | No| Internet Explorer 11|
```

Comment:
Do you need the `&nbsp;` encoding?
```suggestion
|Windows 8.1,<br>Windows 10 ver. < 1903| Microsoft 365 | No| Internet Explorer 11|
```

Reply:
Yes. To be consistent with other cells where it is needed to prevent bad line breaks and in case this cell's contents lengthen it will be needed to prevent a bad line break.